### PR TITLE
Added CI configuration

### PR DESCRIPTION
**Pull request by alexca1703**
Added CI configuration

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v16.16.0
```
Comment (JuanDelahozMiranda) [inline .nvmrc]: Please fix the EOF

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        run: npm ci
+
+      - name: Code style
+        run: npm run lint
+
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v4
+        with:
+          browser: chrome
+          headed: true
```
Comment (JuanDelahozMiranda) [inline .github/workflows/main.yml]: Please fix the EOF

**Review by JuanDelahozMiranda — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (alexca1703) [reply .github/workflows/main.yml]: Fixed, Please review @JuanDelahozMiranda

File: .nvmrc (hunk 1 above)
Comment (alexca1703) [reply .nvmrc]: Fixed, Please review @JuanDelahozMiranda

**Review by JuanDelahozMiranda — APPROVED** [review]

**Pull request merged**